Dockerfile

```diff
@@ -103,6 +103,8 @@ USER ${APP_USER}
 
 RUN gradle openApiGenerate dependencies --no-daemon
```

Comment:
as you renamed the openApiGenerate task into openApiGenerateBFF, you have to fix it also here
```suggestion
RUN gradle openApiGenerateBFF dependencies --no-daemon
```

Reply:
updated